From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: tallbiscuits <112773728+tallbiscuits@users.noreply.github.com>
Date: Tue, 29 Sep 2026 22:18:36 +0200
Subject: [PATCH 02/14] KlippyStatePanel: no Klipper and Moonraker log buttons

Halyard keeps no klippy.log or moonraker.log, so both buttons led to a 404.

diff --git a/src/components/panels/KlippyStatePanel.vue b/src/components/panels/KlippyStatePanel.vue
index 9f0fe1c..5c7ab28 100644
--- a/src/components/panels/KlippyStatePanel.vue
+++ b/src/components/panels/KlippyStatePanel.vue
@@ -22,29 +22,6 @@
                             {{ $t('Panels.KlippyStatePanel.FirmwareRestart') }}
                         </v-btn>
                     </v-col>
-                    <!-- LOG DOWNLOAD BUTTONS -->
-                    <v-col>
-                        <v-btn
-                            :href="apiUrl + '/server/files/klippy.log'"
-                            small
-                            outlined
-                            text
-                            :class="buttonClasses"
-                            @click="downloadLog">
-                            <v-icon class="mr-2">{{ mdiDownload }}</v-icon>
-                            {{ $t('Panels.KlippyStatePanel.KlipperLog') }}
-                        </v-btn>
-                        <v-btn
-                            :href="apiUrl + '/server/files/moonraker.log'"
-                            small
-                            outlined
-                            text
-                            :class="buttonClasses"
-                            @click="downloadLog">
-                            <v-icon class="mr-2">{{ mdiDownload }}</v-icon>
-                            {{ $t('Panels.KlippyStatePanel.MoonrakerLog') }}
-                        </v-btn>
-                    </v-col>
                 </v-row>
             </div>
             <!-- LOADER -->
@@ -87,7 +64,6 @@ import ConnectionStatus from '../ui/ConnectionStatus.vue'
 import Panel from '@/components/ui/Panel.vue'
 import {
     mdiRestart,
-    mdiDownload,
     mdiMessageOutline,
     mdiAlertOutline,
     mdiRocketLaunch,
@@ -102,7 +78,6 @@ import {
 export default class KlippyStatePanel extends Mixins(BaseMixin) {
     mdiPrinter3d = mdiPrinter3d
     mdiRestart = mdiRestart
-    mdiDownload = mdiDownload
     mdiPower = mdiPower
 
     get klippy_message(): string | null {
@@ -155,14 +130,6 @@ export default class KlippyStatePanel extends Mixins(BaseMixin) {
         this.$socket.emit('printer.firmware_restart', {}, { loading: 'firmwareRestart' })
     }
 
-    downloadLog(event: MouseEvent) {
-        event.preventDefault()
-
-        const target = event.target as HTMLElement | null
-        const href = target?.closest('a')?.href ?? ''
-        if (href) window.open(href)
-    }
-
     powerOn() {
         this.$socket.emit(
             'machine.device_power.post_device',
